CHANGELOG.md

```diff
@@ -7,6 +7,10 @@
 * Add elm project type.
 * [#1821](https://github.com/bbatsov/projectile/pull/1821): Add `pyproject.toml` discovery for python projects.
 
+### Changes
+
+* [#1285](https://github.com/bbatsov/projectile/pull/1825): By default, use [fd](https://github.com/sharkdp/fd) in Git repositories instead of `git ls-files` when it is installed, in order to solve the problem where deleted files were still shown in `projectile-find-file` until their deletions were staged. The user-facing behavior should be the same, although potentially with different performance characteristics in large Git repositories. The old behavior can be reclaimed by setting `projectile-git-use-fd` to nil.
```

Comment:
I'd also make a note of this in the docs (probably somewhere in usage and in the FAQ).

Reply:
https://github.com/bbatsov/projectile/pull/1825/commits/e49fee486cbb1e3d8caf446c91d057a7ffcb0f93

Reply:
Just realized there is a separate usage page apart from the readme. It already says fd is used when possible and installed, shall I add more detail about how Projectile determines which tool to use?

Reply:
Yeah, I guess that'd be a nice addition to the docs.

Reply:
https://github.com/bbatsov/projectile/pull/1825/commits/b68f7efe985f6afb0503f056ae8877dc8a77821b